Extensions/draft_saver.js

```diff
@@ -0,0 +1,160 @@
+//* TITLE       Draft Saver **//
+//* DEVELOPER   new-xkit **//
+//* VERSION     1.0.0 **//
+//* DESCRIPTION Saves recent copies of unfinished posts in case your browser crashes **//
+//* FRAME       false **//
+//* SLOW        false **//
+//* BETA        false **//
+
+XKit.extensions.draft_saver = new Object({
+	running: false,
+	api_key: XKit.api_key,
+	interval: null,
+
+	run: function(){
+		this.running = true;
+		XKit.tools.init_css("draft_saver");
+		var number_of_saves = XKit.extensions.draft_saver.preferences.number_of_saves.value;
+		var time_between_saves = XKit.extensions.draft_saver.preferences.number_of_saves.value;
```

Comment:
Ah, good call. Fixed. (And tweaked the spacing issue that was making Travis yell.)